### PR TITLE
VZ-4093: Fix verify-scripts acceptance test

### PR DESCRIPTION
**Pull request by bobdonat**
# Description

This pull request fixes the verify-scripts install acceptance test and removes the flaky test destination.
The changes made were:
1.  don't read the install log to get the installed console URLs but instead get the URLs from the verrazzano resource status block
2. don't use the verrazzano API to get the expected console URLs but instead get the URLs from the ingress resources

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/scripts/install/install_test.go [hunk 1]
```diff
@@ -1,204 +1,126 @@
...
-})
-
-// This test checks that the console output at the end of an install does not show a
-// user URL's that do not exist for that installation platform.  For example, a managed
-// cluster install would not have console URLs.
+// This test checks that the verrazzano install resource has the expected console URLs.
 var _ = Describe("Verify Verrazzano install scripts", func() {
 
-	Context("Verify Console URLs in the install log", func() {
+	Context("Verify Console URLs in the installed verrazzano resource", func() {
```
Comment (markxnelson) [inline tests/e2e/scripts/install/install_test.go:31]: probably should capitalize Verrazzano here

**Review by markxnelson — APPROVED** [review]

File: tests/e2e/scripts/install/install_test.go [hunk 2]
```diff
@@ -1,204 +1,126 @@
...
+		pkg.Log(pkg.Error, fmt.Sprintf("There is an error getting console URLs from ingress resources - %v", err))
 		return false
 	}
 
 	return pkg.SlicesContainSameStrings(consoleUrls, expectedConsoleUrls)
 }
 
-// Get the list of console URLs from the install log, after the line containing "Installation Complete."
-func getConsoleURLsFromLog(installLog string) ([]string, error) {
+// Get the list of console URLs from the status block of the installed verrrazzano resource
```
Comment (davisgomes) [inline tests/e2e/scripts/install/install_test.go:73]: extra "r" in Verrazzano

Comment (markxnelson) [reply tests/e2e/scripts/install/install_test.go:73]: arrrrghhh matey1

File: tests/e2e/scripts/install/install_test.go (hunk 1 above)
Comment (bobdonat) [reply tests/e2e/scripts/install/install_test.go]: fixed

File: tests/e2e/scripts/install/install_test.go (hunk 2 above)
Comment (bobdonat) [reply tests/e2e/scripts/install/install_test.go]: fixed

**Review by markxnelson — APPROVED** [review]

**Pull request merged**